cookie_consent/middleware.py

```diff
@@ -26,7 +31,13 @@ def process_response(self, request, response):
                 continue
             group_version = cookie_dic.get(cookie_group.varname, None)
             for cookie in cookie_group.cookie_set.all():
-                if cookie.name not in request.COOKIES:
+                if cookie.name not in str(request.COOKIES):
+                    continue
+                if group_version == None and not settings.COOKIE_CONSENT_OPT_OUT:
```

Comment:
```suggestion
                if group_version is None and not settings.COOKIE_CONSENT_OPT_OUT:
```

Reply:
Yes you can use 'is' with None. But if you aren't sure which object type you are dealing with, == seems to be a safe choice. 

I am fine with leaving this or changing this depending on what the maintainers want to do.

Reply:
`foo is None` is the pythonic way and `==` goes through `__eq__` which may cuase problems with custom classes. If comparing against the default value of `dict.get` - which returns `None` for missing keys by default, you want identity checks and not equality checks.

Reply:
Interesting. I asked IRC about this and what I commented above is what they told me. I won't get worked up over == or None so I am fine with changing this code to use is.